docs/telemetry/adapters/kafka/index.md

```diff
@@ -0,0 +1,170 @@
+---
+layout: default
+title: "kafka-telegraf"
+date: 2022-03-1 20:03:00 --0600
+categories:
+---
+
+## Introduction
+
+Kafka is a popular message bus system that allows applications to communicate over a pub/sub bus
+as either a publisher or a consumer.  
+A popular method of distributing streaming telemetry is to take the telemetry data and output it
+to a kafka topic so it can be further reacted upon.
+
+This lab will leverage the telegraf container to take streaming telemetry.  
+from two cEOS lab devices from their gNMI interfaces and output the data to a kafka topic.
+
+## Prerequisite
+
+- [containerlab](https://containerlab.dev/)
+- [docker](https://www.docker.com/)
+- [ceos](https://containerlab.dev/manual/kinds/ceos/)
+
+cEOS lab will need to be downloaded from the arista software downloads
+and imported via docker with a tag of 4.29.2F
+
+## Environment
+
+The containerlab file
+<details><summary>Reveal output</summary>
+<p>
+
+```bash
+--8<-- "src/kafka-telegraf/initial.yaml"
+```
+
+</p>
+</details>
+
+Looking at the telegraf.conf file
+
+<details><summary>Reveal output</summary>
+<p>
+
+```bash
+--8<-- "src/kafka-telegraf/telegraf.conf"
+```
+
+</p>
+</details>
+We can see that we are going to have telegraf use the
+[gnmi input plugin](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/gnmi)
+which will simply connect to the two ceos nodes and start to stream their interface counters and bgp statistics.  
+On the outputs portion we can see that we are going to take this gNMI data and send it to the
+kafka broker on the subject of telegraf.
+So any application that connects to the same kafka broker will also be able to see this data.
+This containerlab file will consist of the following Docker containers
+- cEOS (2) each running gNMI interface
```

Comment:
Leave spaces between lists. For example, this is the render in my local serve 
![image](https://user-images.githubusercontent.com/34353177/216170435-75eb120b-8de4-4c92-a39e-434a916cb99f.png)

vs. 

![image](https://user-images.githubusercontent.com/34353177/216170573-bdc212cc-68fa-41f2-aedb-24be8af0ca47.png)